Added Tutorials for Google Batch

zyosufzai opened the pull request:
## Pull Request Template

### Description
<!-- Provide a description of your changes here. If this PR is related to an issue, list the issue number/name here -->
*Added tutorials for using Google Batch based on the LS API Tutorials, added usage of Google Batch in ReadMe, added warnings in the LS API tutorials that the product is depreciated and linked to the Google Batch tutorials, and added warning in Snakemake tutorials that LS API is depreciated and to check Snakemake documentation for when they have instuctions on Google Batch (right now they only use LS API same goes for Cromwell)*

### Assignee
<!-- Mention the GitHub username of the user you want to assign this PR to -->
*Assignees: @kyleoconnell-NIH *

## PR checklist
*Please ensure the following:*
- [X] This comment contains a description of changes (with reason).
- [X] All changes were tested.
- [X] If you've fixed a bug mention the issue number/name.
- [X] Apply approriate tags (e.g. documentation, bug)

kyleoconnell-NIH commented on tutorials/notebooks/GoogleBatch/nextflow/batch_images/service_account_4.jpg:
Can you black out the Principal and the project name?

kyleoconnell-NIH commented on tutorials/notebooks/GoogleBatch/nextflow/batch_images/service_account_1.jpeg:
Block out the project name

kyleoconnell-NIH commented on tutorials/notebooks/GoogleBatch/nextflow/batch_images/Job_l.png:
Block project name

kyleoconnell-NIH commented on tutorials/notebooks/LifeSciencesAPI/snakemake/LS_API_Snakemake.ipynb:
```diff
@@ -523,9 +525,9 @@
   },
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "PySpark (Local)",
```
Why are we using PySpark Kernel?

kyleoconnell-NIH reviewed (changes requested):
Did you also convert the Nextflow Tutorial under LS API? to Batch?

zyosufzai commented:
> Did you also convert the Nextflow Tutorial under LS API? to Batch?

yes I added a Batch tutorials for Nextflow and added a warning in the LS API that it is depreciated and linked back to the Batch tutorials. I didn't remove the LS API tutorials because technically you can still use LS API until 2025

zyosufzai commented on README.md:
```diff
@@ -65,6 +66,9 @@ To spin up a Notebook instance and import an example training notebook, follow o
 If you want to practice using the terminal or review BASH commands in Jupyter, look at [this module](https://github.com/NIGMS/IntroBioinformaticsDartmouth) from Dartmouth developed for the NIGMS Sandbox. 
 
+## **Generative AI in Vertex AI** <a name="genai"></a>
+Vertex AI not only has the capabilities to create Jupyter notebooks but also generative AI models. Some examples of what generative AI models are capabile of are extracting wanted information from text, transforming speech into text, generating images from describtions and vice versa, and so much more. Vertex AI's [Generative AI Studio](https://cloud.google.com/vertex-ai/docs/generative-ai/learn/ ...
```
resolved with the latest push

petersonjdNIH commented on tutorials/notebooks/GenAI/GenAIStudioGCP.ipynb:
@zyosufzai Is this deep link to the Google Cloud Console safe for the public, or is it deep linking to something that only belongs to NIH?

zyosufzai replied:
Do you mean this link 'https://console.cloud.google.com/vertex-ai/generative/language?_ga=2.182664366.923116401.1692009977-1042353744.1691708677'? No this is not NIH specific it is a link to the Generative AI Studio. If you'd like I can instead replace this with instruction on how to get to this page instead (e.g. Generative AI Studio > Language > Article Summary)
